apiserver/start-master.sh

```diff
@@ -8,4 +8,4 @@ echo "0.0.0.0 spark-master" >> /etc/hosts
 
 export SPARK_DAEMON_JAVA_OPTS="-Djava.net.preferIPv4Stack=true"
 
-/spark-2.2.0-bin-hadoop2.7/bin/spark-class org.apache.spark.deploy.master.Master --host spark-master --port 7077 --webui-port 8080
+spark-class org.apache.spark.deploy.master.Master --host spark-master --port 7077 --webui-port 8080
```

Comment:
It appears specifying the full path isn't necessary and avoiding it reduces the number of times we need to change the version string (or number of parameters)
<img width="330" alt="Screenshot 2019-04-10 14 04 04" src="https://user-images.githubusercontent.com/5543229/55902642-f92b5000-5b99-11e9-8c08-c7ecb0b05568.png">
<img width="677" alt="Screenshot 2019-04-10 14 04 34" src="https://user-images.githubusercontent.com/5543229/55902643-f9c3e680-5b99-11e9-9919-9b2e1b7e1fed.png">